openstack/datadog_checks/openstack/openstack.py

```diff
@@ -886,8 +861,8 @@ def get_all_servers(self, i_key, collect_all_tenants, filter_by_host=None):
         url = '{0}/servers/detail'.format(self.get_nova_endpoint())
         headers = {'X-Auth-Token': self.get_auth_token()}
 
-        if collect_all_tenants:
-            query_params["all_tenants"] = True
+        # Note this query param specifically requires admin user rights
```

Comment:
Let's remove the param, then?

Reply:
Apologies, what I meant was that it doesn't make sense to make it configurable since we want to batch get everything and this query_param allows that. So I removed the yaml config option, but the query param allows us to batch query for all servers belonging to all projects at once.

Reply:
ah got it